.github/workflows/ci.yml

```diff
@@ -1,17 +1,11 @@
 name: ci
 on:
   push:
-    paths:
-      - '**.go'
-      - '**.mod'
-      - '**.sum'
     branches:
-      - master
+      - main
   pull_request:
-    paths:
-      - '**.go'
-      - '**.mod'
-      - '**.sum'
+      branches:
+      - main
```

Comment:
I think the reason the CI isn't running now is because this part of the workflow spec will only run for pull requests that have a branch named `main`. I think what you want instead is:

```yaml
on:
    push:
        branches:
          - main
     pull_request: {}
```